pkg/netpol/connlist/connlist.go

```diff
@@ -494,8 +532,8 @@ func (ca *ConnlistAnalyzer) warnBlockedIngress(peerStr string, ingressObjs map[s
 		warningMsg = "Route resource " + ingressObjs[scan.Route][0]
```

Comment:
warning messages should start with capital letters consistently

Reply:
here this is not the beginning of the warning message 
checking for other warning messages